rosa_cluster_admin/rosa-configuring-pid-limits.adoc

```diff
@@ -21,6 +21,8 @@ A process identifier (PID) is a unique identifier assigned by the Linux kernel t
 
 In {product-title} 4.11 and later, by default, a pod can have a maximum of 4,096 PIDs. If your workload requires more than that, you can increase the allowed maximum number of PIDs by configuring a `KubeletConfig` object.
 
+{product-title} clusters running on earlier versions use a default PID limit of `1024`.
```

Comment:
"running _on_ earlier versions" seems like an error; did you mean "running earlier versions"? I also think that because this is its own paragraph, it needs to be more specific re: what "earlier versions" means.

```suggestion
{product-title} clusters running versions earlier than 4.11 use a default PID limit of `1024`.
```

Note: I just realized that the snippet itself uses this text and might need to be updated, too (the "running on earlier versions" part). However, the specificity issue isn't present in the snippet because it's all in one paragraph.